hack/make/tgz

```diff
@@ -9,6 +9,11 @@ if [ ! -d "$CROSS/linux/amd64" ]; then
 	false
 fi
 
+bundle .integration-daemon-start
+bundle .detect-daemon-osarch
+
+make manpages
```

Comment:
I don't think there is any more, as man pages have their own Dockerfiles. cc @dnephin is there a better way?